clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/v1/api/KubernetesClientApiConverter.groovy

```diff
@@ -964,7 +964,11 @@ class KubernetesClientApiConverter {
     return daemonset
   }
 
-  static boolean canUpdated(DeployKubernetesAtomicOperationDescription description) {
+  static Boolean canUpdated(DeployKubernetesAtomicOperationDescription description) {
     return description.updateController?.enabled
   }
+
+  static Boolean checkSequence(DeployKubernetesAtomicOperationDescription description) {
```

Comment:
What does it mean to check the sequence? It seems to check that it's non negative. Can you update the function name?